idea/src/org/jetbrains/kotlin/idea/inspections/ReplaceNegatedIsEmptyWithIsNotEmptyInspection.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.inspections
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemHighlightType
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiElementVisitor
+import org.jetbrains.kotlin.idea.core.replaced
+import org.jetbrains.kotlin.idea.inspections.collections.isCalling
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.kotlin.psi.psiUtil.getLastParentOfTypeInRow
+
+class ReplaceNegatedIsEmptyWithIsNotEmptyInspection : AbstractKotlinInspection() {
+    override fun buildVisitor(holder: ProblemsHolder, isOnTheFly: Boolean): PsiElementVisitor {
+
+        return simpleNameExpressionVisitor { simpleNameExpression ->
+            if (simpleNameExpression.isEmptyNegation()) {
+                holder.registerProblem(
+                    simpleNameExpression,
```

Comment:
Please check highlighting range here when inspection's level is increased manually. Too wide highlighting range is bad. I think here we should highlight just `isEmpty` reference, not the whole `!someVeryImportantList.isEmpty()`

Reply:
Thank you.
I have confirmed that only isEmpty reference is highlighted.

<img width="362" alt="2018-11-17 9 28 47" src="https://user-images.githubusercontent.com/744059/48654021-c945a980-ea4c-11e8-89d2-57a8d4ecfd5b.png">